test/pseudo-tty/test-assert-no-color.js

```diff
@@ -17,3 +17,26 @@ assert.throws(
       '-   foo: \'bar\'\n' +
       '- }\n',
   });
+
+{
+  // TODO(puskin94): remove the emitWarning override once the partialDeepStrictEqual method is not experimental anymore
+  // Suppress warnings, necessary otherwise the tools/pseudo-tty.py runner will fail
+  const originalEmitWarning = process.emitWarning;
+  process.emitWarning = common.mustCall();
```

Comment:
If this is only meant to suppress the warning output, can't you use the `--no-warnings` flag instead?

Reply:
@jasnell because I keep forgetting I can define "per file" flags, thanks for the hint 😄